Source/VFECore/Genes/Harmony/PawnRenderer_GetBodyOverlayMeshSet.cs

```diff
@@ -56,10 +56,10 @@ public static class PawnRenderer_BaseHeadOffsetAt_Patch
 
         static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions)
         {
-            var headOffset = AccessTools.Field("BodyTypeDef:headOffset");
-            var pawn = AccessTools.Field("PawnRenderer:pawn");
-            var bodyScaleFactor = AccessTools.Method("PawnRenderer_BaseHeadOffsetAt_Patch:LifeStageFactorUpdated");
-            var codes = instructions.ToList();
+            FieldInfo headOffset = AccessTools.Field(typeof(BodyTypeDef), nameof(BodyTypeDef.headOffset));
+            FieldInfo pawn = AccessTools.Field(typeof(PawnRenderer), "pawn");
+            MethodInfo bodyScaleFactor = ((Func<Vector2, Pawn, Vector2>)LifeStageFactorUpdated).Method;
+            List<CodeInstruction> codes = instructions.ToList();
```

Comment:
```suggestion
            var codes = instructions.ToList();
```
Why did you even change this line?